Source/Xamarin/Prism.Forms/PrismApplicationBase.cs

```diff
@@ -1,15 +1,23 @@
 ﻿using Prism.Logging;
 using Prism.Modularity;
 using Prism.Navigation;
+using System.Linq;
 using Xamarin.Forms;
 #if TEST
 using Application = Prism.FormsApplication;
 #endif
 
 namespace Prism
 {
-    public abstract class PrismApplicationBase : Application
+    public abstract class PrismApplicationBase<T> : Application
     {
+        IPlatformInitializer<T> _platformInitializer;
+
+        /// <summary>
+        /// The dependency injection container used to resolve objects
+        /// </summary>
+        public T Container { get; protected set; }
```

Comment:
Not worried about that at all. Ninject is still a container. Also, this will make everything consistent with docs and samples. API will be the same no matter what container you use.